### ISSUE TITLE
FluidBufSTFT: Loading results buffers in sclang when win > 1024

### ISSUE THREAD

**Issue by weefuzzy** [flucoma/flucoma-sc#22]
via https://discourse.flucoma.org/t/fluidbufstft-windowsizes-above-1024-dont-work/953 

`Buffer#loadToFloatArray` fails to work with window sizes greater than 1024. The tmp file doesn't seem to get written, so perhaps there's an upper channel limit that's failing in scsynth. 

`getToFloatArray` works, so the process itself seems to be ok (although streaming caps the size).

**Comment by weefuzzy** [comment]
This seems to be built into libsndfile (https://github.com/libsndfile/libsndfile/pull/468), so I guess the best we'll be able to manage is a workaround either 
* finding some way of writing buffers to an alternative format 
* streaming in chunks with `getToFloatArray`

**Comment by tedmoore** [comment]
tested it just confirm / see where this was. Thought I would drop my test code here for any curious lurkers.

```
b = Buffer.read(s,FluidFilesPath("Nicol-LoopE-M.wav"));

m = Buffer(s);

FluidBufSTFT.processBlocking(s,b,magnitude:m,windowSize:1024,action:{"done".postln});

(
// works fine
m.loadToFloatArray(action:{
	arg mags;
	mags.postln;
});
)

// do it again, but with windowSize > 1024
FluidBufSTFT.processBlocking(s,b,magnitude:m,windowSize:2048,action:{"done".postln});

(
// fails
m.loadToFloatArray(action:{
	arg mags;
	mags.postln;
});
)

(
// works fine (takes a second though)
m.getToFloatArray(action:{
	arg mags;
	mags.postln;
});
)

```

**Comment by weefuzzy** [comment]
Yup, that's the state of it. I guess we should document this, but it's certainly not going to change at libsndfile's end, which makes it pretty unlikely to change from scsynth's end too.

**Comment by tedmoore** [comment]
Sounds good. I'll add documenting this to milestone beta5.

**Comment by tedmoore** [comment]
To be closed when https://github.com/flucoma/flucoma-docs/pull/56 is merged.
